core/src/main/java/org/mifos/mobilewallet/core/domain/model/CheckBoxStatus.java

```diff
@@ -0,0 +1,27 @@
+package org.mifos.mobilewallet.core.domain.model;
```

Comment:
UI models should not be a part of core module. Core module should only have the usecase related entities

Reply:
Okay I will correct it